lnwallet/commitment.go

```diff
@@ -1180,7 +1182,7 @@ func addHTLC(commitTx *wire.MsgTx, ourCommit bool,
 
 	// Store the pkScript of this particular PaymentDescriptor so we can
 	// quickly locate it within the commitment transaction later.
-	if ourCommit {
+	if whoseCommit == lntypes.Local {
```

Comment:
very thin API/atypical for enums, but all these equality checks tempt me to add `IsLocal()` and `IsRemote` to the `ChannelPartyType`

Reply:
Happy to add it. I've had this thought too.